platform/linux-generic/odp_queue.c

```diff
@@ -584,8 +556,9 @@ static int queue_init(queue_entry_t *queue, const char *name,
 	queue->s.pktin = PKTIN_INVALID;
 	queue->s.pktout = PKTOUT_INVALID;
 
-	queue->s.head = NULL;
-	queue->s.tail = NULL;
+	ring_st_init(&queue->s.ring_st,
+		     queue_tbl->ring_data[queue->s.index].data,
+		     CONFIG_QUEUE_SIZE);
```

Comment:
Must use `CONFIG_QUEUE_SIZE - 1` here, as noted earlier, if we're not going to used the user-supplied queue size.

Reply:
One entry is not lost.